src/screens/HomeScreen.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+
+import { useNavigation } from '@react-navigation/native';
+import { StackNavigationProp } from '@react-navigation/stack';
+import { ScrollView, StyleSheet, View } from 'react-native';
+import { Button, Text, TextInput } from 'react-native-paper';
+
+import Title from '../components/Title';
+import Screen from '../components/layout/Screen';
+import { Colors } from '../constants';
+import { Routes } from '../routes/routes';
+import { RouteParams } from '../routes/types';
+import Subtitle from '../components/Subtitle';
+import { MyContext } from "../store/myStore";
+
+type RoutePropType = StackNavigationProp<RouteParams, Routes.SignUpCongrats>;
+type UserProfile = {
+    name: string;
+    sex: string;
+    height: number;
+    weight: number;
+    age: number;
+    activity: string;
+    goal: string;
+    target: number;
+    userId: number;
+};
```

Comment:
Is better to move the shared type into a different file. Now you don't reuse them, but we both know that this will be needed in future :)